Packs/Carbon_Black_Enterprise_Response/TestPlaybooks/playbook-Carbon_Black_Response_Test.yml

```diff
@@ -1,7 +1,7 @@
 id: Carbon Black Response Test
 version: -1
 name: Carbon Black Response Test
-description: Tests carbon black enterprise response selected use cases and commands
+description: Deprecated. No available replacement.
```

Comment:
```suggestion
description: Deprecated. No available replacement because "carbonblack v2" is deprected. Use "VMware Carbon Black EDR v2" integration instead.
```